src/SprykerEco/Zed/Payone/Communication/Controller/GatewayController.php

```diff
@@ -184,4 +186,24 @@ public function getPaymentDetailAction(PayoneGetPaymentDetailTransfer $getPaymen
         $getPaymentDetailTransfer->setPaymentDetail($response);
         return $getPaymentDetailTransfer;
     }
+
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return \Generated\Shared\Transfer\AddressCheckResponseTransfer
+     */
+    public function sendAddressCheckRequestAction(QuoteTransfer $quoteTransfer): AddressCheckResponseTransfer
+    {
+        return $this->getFacade()->sendAddressCheckRequest($quoteTransfer);
```

Comment:
I'm wondering how it works if method in your facade has different name.

Reply:
????